include/clang/Interpreter/CppInterOp.h

```diff
@@ -15,6 +15,13 @@
 #include <string>
 #include <vector>
 
+// The cross-platform DllExport macro definition
+#ifdef _WIN32
+#define DllExport __declspec(dllexport)
+#else
+#define DllExport __attribute__((visibility("default")))
+#endif
```

Comment:
```suggestion
#ifdef _WIN32
#define CPPINTEROP_API __declspec(dllexport)
#else
#define CPPINTEROP_API __attribute__((visibility("default")))
#endif
```

It's probably better name.